.travis.yml

```diff
@@ -1,18 +1,12 @@
 language: go
 go:
-- '1.10.x'
-
-
-cache:
-  directories:
-    - $GOPATH/src/gx
+- '1.13.x'
 
 install:
-  - go get github.com/whyrusleeping/gx
-  - go get github.com/whyrusleeping/gx-go
-  - gx install --global
-  - gx-go rw
+  - GO111MODULE=on
```

Comment:
check cluster's travis file and do things similarly (i.e. using `env` for env variables or caching the modules)